core/templates/domain/classroom/classroom-backend-api.service.ts

```diff
@@ -41,6 +41,48 @@ interface ClassroomDataBackendDict {
   'topic_list_intro': string;
 }
 
+interface ClassroomIdToClassroomNameBackendDict {
+  'classroom_id_to_classroom_name': {
+    [classroomId: string]: string;
+  };
+}
+
+interface ClassroomIdToClassroomNameResponse {
+  [classroomId: string]: string;
+}
+
+interface NewClassroomIdBackendDict {
+  'classroom_id': string;
+}
+
+interface ClassroomDict {
```

Comment:
This should be `ClassroomBackendDict` instead of `ClassroomDict`.

Reply:
Done